### PR TITLE
refactor(TG-309): angular2 template loader

### PR DESCRIPTION
**Pull request by igor0ser**
changing from template, styles to templateUrl, stylesUrl

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/156" target="_blank">ReviewNinja

File: src/app/dashboard/dashboard-editor/dashboard-editor.component.ts [hunk 1]
```diff
@@ -4,14 +4,14 @@ import { DragulaService } from 'ng2-dragula/ng2-dragula';
 import { DashboardService } from '../dashboard.service';
 import { Widget } from '../widget.model';
 
-const template = require('./dashboard-editor.template.html');
-const style = require('../dashboard.style.scss');
+import { DashboardService } from '../dashboard.service';
```
Comment (velmyk) [inline src/app/dashboard/dashboard-editor/dashboard-editor.component.ts]: !resolve This was imported twice

File: src/app/dashboard/sensor-executor-widget/sensor-executor-widget.component.ts [hunk 2]
```diff
@@ -7,13 +7,10 @@ import {
...
 import { SensorWidgetService } from '../shared/sensor-widget/sensor-widget.service';
 
-const template = require('./sensor-executor-widget.template.html');
-const styles = require('../shared/sensor-widget/sensor-widget.style.scss');
-
 @Component({
     selector: 'sm-sensor-executor-widget',
-    template,
-    styles: [styles]
+    templateUrl: './sensor-executor-widget.template.html',
```
Comment (velmyk) [inline src/app/dashboard/sensor-executor-widget/sensor-executor-widget.component.ts:12]: Can we use moduleId property in decorator not to write absolute path?

File: src/app/login/login.component.ts [hunk 3]
```diff
@@ -3,14 +3,12 @@ import { Router } from '@angular/router';
 import { Subscription } from 'rxjs/Rx';
 import { AuthService } from '../core/auth/auth.service';
 import { IUserCredentials } from '../core/auth/auth.interfaces';
-
-const styles = require('./login.component.scss');
-const template = require('./login.component.html');
+import { Subscription } from 'rxjs/Rx';
```
Comment (velmyk) [inline src/app/login/login.component.ts]: !resolve You import it twice

Comment (piton13) [inline src/app/login/login.component.ts]: duplicated line

**Comment by borys-ihnatyev** [comment]
Did you add the loader itself ? https://github.com/TheLarkInn/angular2-template-loader

File: src/app/dashboard/sensor-executor-widget/sensor-executor-widget.component.ts (hunk 2 above)
Comment (igor0ser) [inline src/app/dashboard/sensor-executor-widget/sensor-executor-widget.component.ts:12]: we are using angular-template-loader, which does all imports behind the scenes

**Comment by igor0ser** [comment]
as it turned out we already had this loader in our build

File: src/app/login/login.component.ts (hunk 3 above)
Comment (igor0ser) [inline src/app/login/login.component.ts]: !resolved

File: src/app/dashboard/dashboard-editor/dashboard-editor.component.ts (hunk 1 above)
Comment (igor0ser) [inline src/app/dashboard/dashboard-editor/dashboard-editor.component.ts]: !resolved

**Pull request merged**